src/azure-cli/azure/cli/command_modules/backup/custom.py

```diff
@@ -663,12 +823,14 @@ def restore_disks(cmd, client, resource_group_name, vault_name, container_name,
         trigger_restore_properties.region = azure_region
         result = crr_client.trigger(azure_region, crr_access_token, trigger_restore_properties, raw=True,
                                     polling=False).result()
+
         return _track_backup_crr_job(cmd.cli_ctx, result, azure_region, vault.id)
 
     # Trigger restore
     result = client.trigger(vault_name, resource_group_name, fabric_name,
                             container_uri, item_uri, rp_name,
-                            trigger_restore_request, raw=True)
+                            trigger_restore_request, raw=True, polling=False).result()
```

Comment:
Yeah, Himanshu. In the latest SDK it seems they've changed the return type of this function call. We need to specify this otherwise it gives an error.